### PR TITLE
[Feature] Minor fixes to liquidity page

### PR DESCRIPTION
**Pull request by berteotti**
### WIP 
Me and @Diogomartf  are working on updating the design of liquidity page and also the fonts and colors.

This is **work in progress** it's not finished, but we want to divide the work in smaller chunks and do things incrementally. 

We will do this smaller PRs and merge it on a "main" branch, `feature/fonts-and-colors-liquidity-page`.

---

# Summary

Fix styles and add translations in liquidity page

### CODE REVIEW

File: public/locales/en.json [hunk 1]
```diff
@@ -135,8 +135,9 @@
   "poolShare": "Pool share",
   "showMore": "Show more",
   "pair": "Pair",
-  "capmings": "Campings",
+  "campaings": "Campaings",
```
Comment (adamazad) [inline public/locales/en.json:138]: Am I reading this right? campings or campaigns?

File: public/locales/en.json [hunk 2]
```diff
@@ -135,8 +135,9 @@
...
   "24hVolume": "24h Volume",
   "farming": "Farming",
-  "carrot": "Carrot"
+  "carrot": "Carrot",
+  "myPairs": "My pairs"
```
Comment (adamazad) [inline public/locales/en.json:142]:
```suggestion
  "myPairs": "My Pairs"
```

File: src/pages/Pools/index.tsx [hunk 3]
```diff
@@ -87,6 +96,7 @@ function Title({
   sortBy,
 }: TitleProps) {
   const [openTokenModal, setOpenTokenModal] = useState(false)
+  const { t } = useTranslation()
```
Comment (adamazad) [inline src/pages/Pools/index.tsx:99]: It makes sense to scope this if you like it keep translations clean

**Review by adamazad — CHANGES_REQUESTED** [review]

File: public/locales/en.json (hunk 1 above)
Comment (berteotti) [reply public/locales/en.json]: Fixed, thanks!

File: public/locales/en.json (hunk 2 above)
Comment (berteotti) [reply public/locales/en.json]: Fixed, thanks!

File: src/pages/Pools/index.tsx (hunk 3 above)
Comment (berteotti) [reply src/pages/Pools/index.tsx:99]: I think we should first define how we scope translations, so everyone is aligned with the scoping strategy.

**Review by Diogomartf — APPROVED** [review]

**Pull request merged**